starters/next-drupal-starter/pages/preview-error/[[...alias]].jsx

```diff
@@ -0,0 +1,43 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import Layout from '../../components/layout';
+
+import {
+	getCurrentLocaleStore,
+	globalDrupalStateStores,
+} from '../../lib/stores';
+
+export default function PreviewError({ footerMenu, preview }) {
+	const { query } = useRouter();
+	return (
+		<Layout footerMenu={footerMenu} preview={true}>
+			<div className="flex flex-col mx-auto prose-xl mt-20 w-1/2">
+				<h2 className="text-center">
+					🛑 {query.error ? query.error : 'There was an error on the server'} 🛑
+				</h2>
```

Comment:
I think we should put the details in a message query string and put that in a `p` tag for readability. The error and message can be destructured from `useRouter` as well.
```suggestion
				<h2 className="text-center">
					🛑 {error ? error : 'There was an error on the server'} 🛑
				</h2>
				<p>{message}</p>
```